code/src/utest/pcgen/base/formula/SubtractingFormulaTest.java

```diff
@@ -56,30 +60,34 @@ public void testEquality()
 		assertFalse(f1.equals(f4));
 	}
 
+	@Test
 	public void testPositive()
 	{
 		SubtractingFormula f = new SubtractingFormula(3);
-		assertEquals(2, f.resolve(Integer.valueOf(5)).intValue());
-		assertEquals(2, f.resolve(Double.valueOf(5.5)).intValue());
+		assertEquals(2, f.resolve(5).intValue());
+		assertEquals(2, f.resolve(5.5).intValue());
 		testBrokenCalls(f);
 	}
 
+	@Test
 	public void testZero()
 	{
 		SubtractingFormula f = new SubtractingFormula(0);
-		assertEquals(5, f.resolve(Integer.valueOf(5)).intValue());
-		assertEquals(2, f.resolve(Double.valueOf(2.3)).intValue());
+		assertEquals(5, f.resolve(5).intValue());
+		assertEquals(2, f.resolve(2.3).intValue());
 		testBrokenCalls(f);
 	}
 
+	@Test
 	public void testNegative()
 	{
 		SubtractingFormula f = new SubtractingFormula(-2);
-		assertEquals(7, f.resolve(Integer.valueOf(5)).intValue());
-		assertEquals(-4, f.resolve(Double.valueOf(-6.7)).intValue());
+		assertEquals(7, f.resolve(5).intValue());
+		assertEquals(-4, f.resolve(-6.7).intValue());
 		testBrokenCalls(f);
 	}
 
+	@Test
```

Comment:
I don't think this should be a test... it's a utility function and probably should be renamed to not start with "test"